File: How_To_Program_Exercises/JavaHowToProgram9/Chapter3/HeartRateCalculatorClass/HeartRatesTest.java
//
// HeartRates Class Test
//

import java.util.Scanner;

class HeartRatesTest
{
	public static void main (String[] args)
	{
		String fName, lName;
		int birthDay, birthMonth, birthYear, currDay, currMonth, currYear;
		
		Scanner input = new Scanner (System.in);
		
		System.out.print("\nEnter your first name: ");
		fName = input.next();
		System.out.print("\nEnter your last name: ");
		lName = input.next();
		
		//Prompt birthdate
		System.out.print("\nEnter your birth date: ");
		birthDay = input.nextInt();
		System.out.print("Enter your birth month: ");
		birthMonth = input.nextInt();
		System.out.print("Enter your birth year: ");
		birthYear = input.nextInt();
		
		//Prompt current date
		System.out.print("\nEnter today's date: ");
		currDay = input.nextInt();
		System.out.print("Enter today's month: ");
		currMonth = input.nextInt();
		System.out.print("Enter today's year: ");
		currYear = input.nextInt();
		
		//Instantiate instance of HeartRates
		HeartRates heartRate = new HeartRates(fName, lName, birthDay, birthMonth, birthYear);
	
		//Display information
		System.out.println("\n\nYour information:\n-------------------");
		System.out.printf("Name: %s %s\n", heartRate.getFirstName(), heartRate.getLastName());
		System.out.printf("Age: %d", heartRate.getAge(currDay, currMonth, currYear));
		System.out.printf("\nMaximum Heart Rate: %d", heartRate.getMaximumHeartRate());
		System.out.printf("\nTarget Heart Rate: %d\n\n", heartRate.getTargetHeartRate());
	
	}
}

File: How_To_Program_Exercises/Visual_C#_How_To_Program2012_Fifth/Chapter4/HeartRatesCalculator/HeartRatesTest.cs
//
// HeartRatesTest Class
//
using System;
class HeartRatesTest
{
	public static void Main (string[] args)
	{
		string fName, lName;
		
		int birthYear, currYear;
		
		Console.Write("\nEnter your first name: ");
		fName = Console.ReadLine();
		Console.Write("\nEnter your last name:");
		lName = Console.ReadLine();
		
		//Prompt birth year
		Console.Write("\nEnter your birth year: ");
		birthYear = Convert.ToInt32(Console.ReadLine());
		
		//Prompt current year
		Console.Write("\nEnter the current year: ");
		currYear = Convert.ToInt32(Console.ReadLine());
		
		HeartRates heartRate = new HeartRates (fName, lName, birthYear, currYear);
	
		Console.WriteLine("\n\nYour information\n------------------");
		Console.WriteLine("Name: {0} {1}", heartRate.FirstName, heartRate.LastName);
		Console.WriteLine("Age: {0}", heartRate.Age);
		Console.WriteLine("Maximum Heart Rate: {0}\nTarget Heart Rate: {1}", 
							heartRate.MaximumHeartRate, heartRate.TargetHeartRate);
	}
}